tls/s2n_kem.c

```diff
@@ -77,6 +78,18 @@ const struct s2n_kem s2n_sike_p434_r2 = {
         .decapsulate = &SIKE_P434_r2_crypto_kem_dec,
 };
 
+const struct s2n_kem s2n_kyber_512_r2 = {
+        .name = "kyber512r2-KEM",
```

Comment:
I'm not sure if we're including the "-KEM" in the name. Will need to double-check this.

Reply:
NOT CHANGED.

Reply:
It is changed as kyber512r2.